### PR TITLE
refactor: initial refactor of onboarding network views

### LINKED ISSUE #654: ️🛠 - Refactor onboarding layout into modal layout
### Task description

Refactor existing onboarding layout and onboarding screens for rebranded layout

### Requirements

- Change onboarding layout
- Update all onboarding screens for new layout

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/onboarding/views/create-profile/views/ChooseCreateProfileFlowView.svelte [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
-        isBusy = { ...isBusy, [createProfileType]: true }
-        const type = createProfileType === CreateProfileType.Ledger ? ProfileType.Ledger : ProfileType.Software
-        updateOnboardingProfile({ createProfileType, type })
+    let selectedCreateProfileType: CreateProfileType | undefined = undefined
+    function onProfileTypeClick(createProfileType: CreateProfileType): void {
+        selectedCreateProfileType = createProfileType
+    }
+
+    async function onContinueClick(): Promise<void> {
+        busyMap = { ...busyMap, [selectedCreateProfileType]: true }
```
Comment (MarkNerdi) [inline packages/desktop/views/onboarding/views/create-profile/views/ChooseCreateProfileFlowView.svelte]:
```suggestion
        busyMap[selectedCreateProfileType] = true
```

File: packages/desktop/views/onboarding/views/restore-profile/views/ChooseRestoreProfileFlowView.svelte [hunk 2]
```diff
@@ -9,28 +8,35 @@
...
+    function onProfileTypeClick(restoreProfileType: RestoreProfileType): void {
+        selectedRestoreProfileType = restoreProfileType
+    }
 
-    async function onProfileTypeClick(restoreProfileType: RestoreProfileType): Promise<void> {
-        isBusy = { ...isBusy, [restoreProfileType]: true }
-        const type = restoreProfileType === RestoreProfileType.Ledger ? ProfileType.Ledger : ProfileType.Software
-        updateOnboardingProfile({ type, restoreProfileType })
+    async function onContinueClick(): Promise<void> {
+        busyMap = { ...busyMap, [selectedRestoreProfileType]: true }
```
Comment (MarkNerdi) [inline packages/desktop/views/onboarding/views/restore-profile/views/ChooseRestoreProfileFlowView.svelte]:
```suggestion
        busyMap[selectedRestoreProfileType] = true
```

File: packages/desktop/views/update-stronghold/views/UpdateStrongholdView.svelte [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
         try {
-            isBusy = true
+            busy = true
             if (isRecovery) {
                 await migrateStrongholdFromOnboardingProfile(password)
             } else {
                 await migrateStrongholdFromActiveProfile(password)
             }
-            isBusy = false
+            busy = false
```
Comment (MarkNerdi) [inline packages/desktop/views/update-stronghold/views/UpdateStrongholdView.svelte]: move `busy = false` into finally + remove `return` in catch

File: packages/desktop/views/onboarding/views/create-from-ledger/views/InstallLedgerView.svelte [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
     import { createFromLedgerRouter } from '../create-from-ledger-router'
 
     function onContinueClick(): void {
         $createFromLedgerRouter.next()
     }
 
+    let isBusy: boolean = false
     async function onBackClick(): Promise<void> {
+        isBusy = true
         await stopPollingLedgerNanoStatus()
```
Comment (MarkNerdi) [inline packages/desktop/views/onboarding/views/create-from-ledger/views/InstallLedgerView.svelte]: do we need a trycatch around this await?

Comment (nicole-obrien) [reply packages/desktop/views/onboarding/views/create-from-ledger/views/InstallLedgerView.svelte]: it wasn't even async lol

File: packages/desktop/views/onboarding/components/OnboardingSelectorTile.svelte [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    export let iconWidth: string = undefined
+    export let disabled: boolean = false
+    export let hidden: boolean = false
+    export let selected: boolean = false
+
+    export let onClick: () => unknown
+</script>
+
+{#if !hidden}
+    <GridTile width="full" variant="outlined" {onClick} {disabled} {selected}>
```
Comment (MarkNerdi) [inline packages/desktop/views/onboarding/components/OnboardingSelectorTile.svelte:19]: the border color shouldnt be black, but `gray-200`. Lets create a task for that in the ui kit

Comment (MarkNerdi) [reply packages/desktop/views/onboarding/components/OnboardingSelectorTile.svelte:19]: Also on `select`, the border width changes to `2px`, therefore the tile content shifts a bit

File: packages/desktop/views/onboarding/views/complete-onboarding/views/EnterPinView.svelte
Comment (MarkNerdi) [inline packages/desktop/views/onboarding/views/complete-onboarding/views/EnterPinView.svelte:1]: can we already use the pininput from the UI kit here?

File: packages/desktop/views/onboarding/components/OnboardingSelectorTile.svelte (hunk 5 above)
Comment (nicole-obrien) [reply packages/desktop/views/onboarding/components/OnboardingSelectorTile.svelte:19]: To be made consisted in bloom-ui

File: packages/desktop/views/onboarding/views/complete-onboarding/views/EnterPinView.svelte
Comment (nicole-obrien) [reply packages/desktop/views/onboarding/views/complete-onboarding/views/EnterPinView.svelte:1]: Waiting to for redesigns to go through full contents of each onboarding screen

**Review by MarkNerdi — APPROVED** [review]

**Pull request merged**